mmdet/models/backbones/detectors_resnet.py

```diff
@@ -0,0 +1,190 @@
+import torch.utils.checkpoint as cp
+from mmcv.cnn import build_conv_layer, constant_init
+
+from ..builder import BACKBONES
+from .resnet import Bottleneck as _Bottleneck
+from .resnet import ResNet
+
+
+class Bottleneck(_Bottleneck):
+    expansion = 4
```

Comment:
Please add standard docstrings for newly added modules and functions. MMDet V2.0 has more strict requirements on the docstrings.